src/Uri.php

```diff
@@ -690,7 +701,13 @@ private function validateState()
                 throw new \InvalidArgumentException('A relative URI must not have a path beginning with a segment containing a colon');
             }
         } elseif (isset($this->path[0]) && $this->path[0] !== '/') {
-            throw new \InvalidArgumentException('The path of a URI with an authority must start with a slash "/" or be empty');
+            @trigger_error(
```

Comment:
It is, because Symfony uses the PHPUnitBridge component to catch deprecation notices in Symfony Full Stack projects.

`E_USER_DEPRECATED` is just a notice, not an error, and will not halt program execution. It will also not be visible on properly configured production systems. (if someone has enabled `E_ALL` in their live error reporting, well… :))

Users not using the PHPUnitBridge or similar components will never see that the method is deprecated and thus not change their calls. Yes, the triggered notice might break the build pipeline, but it will not break running projects.

I am not saying that I would insist on removing the `@`, it's just that `E_USER_DEPRECATED` is meant to be used. Muting it so that e.g. PHPUnitBridge can make something out of it means making an agnostic library focusing on specific environments.